### PR TITLE
Fixes #812 : Added privacy page

### LINKED ISSUE #812: Adding "PRIVACY" Page and linking it in the Footer
**Current behavior:**
There is no page for privacy policy in the footer.

**Expected behavior:**
There should be a link to the "PRIVACY" page in the footer on the right hand side.

**Would you like to work on this issue?**
Yes.

### PR DESCRIPTION
**Pull request by marvin08**
#### Changes proposed in this pull request:
I have added the privacy policy page of susper and linked it to the footer.

screenshot from 2017-11-15 22 16 27

screenshot from 2017-11-15 22 20 30

Fixes #812.

### CODE REVIEW

**Comment by marvin08** [comment]
@harshit98 Please review.

**Comment by harshit98** [comment]
@marvin08 Why you are including `package-lock.json` file ?

**Comment by marvin08** [comment]
@harshit98 I think it was installed alongside node_modules.

**Comment by harshit98** [comment]
@marvin08 Please remove it. 👍

**Comment by marvin08** [comment]
@harshit98 Done. :+1:

File: src/app/privacy/privacy.component.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+a:hover {
+	color: rgb(51,51,51);
+	text-decoration: none;
+}
+
+.navbar-brand {
+	padding-left: 100px;
+}
```
Comment (aashutoshrathi) [inline src/app/privacy/privacy.component.css]: Add a newline at the end.

Comment (marvin08) [reply src/app/privacy/privacy.component.css]: Done. :+1:

**Comment by aashutoshrathi** [comment]
Squash all 3 commits

**Comment by marvin08** [comment]
@harshit98 I am closing this PR as something went wrong while squashing the commits.

**Comment by marvin08** [comment]
@harshit98 I have closed this as something went wrong with the commits.

**Pull request closed without merging**